feed.go

```diff
@@ -441,7 +441,7 @@ func (feed *Feed) parseAgencies(path string, prefix string) (err error) {
 			}
 
 			if len(existingAgId) > 0 && feed.Agencies[existingAgId].Timezone != agency.Timezone {
```

Comment:
I have force pushed to add .String() to both sides since it then just compares the name of each timezone.
If I had to guess this may be caused by either us comparing two * values (maybe that compares only the references and not the underlying data?) or by something else in the Location struct being different which we do not care about.